bentoml/_internal/cli/model_management.py

```diff
@@ -54,16 +54,27 @@ def model_cli():
     @click.option(
         "-o",
         "--output",
-        type=click.Choice(["tree", "json", "yaml", "path"]),
-        default="tree",
+        type=click.Choice(["json", "yaml", "path"]),
+        default="yaml",
     )
     def get(model_tag, output):
         """Print Model details by providing the model_tag
 
         bentoml model get FraudDetector:latest
         bentoml model get FraudDetector:20210709_DE14C9
         """
-        pass
+        res = model_store.get(model_tag).info
+        console = Console()
+
+        if output == "path":
+            path = model_store._fs.getsyspath(res.tag.path())
```

Comment:
Maybe just store the model get and use `Model.path()`? This doesn't actually work for version prefixes. Same below.